process_config.py

```diff
@@ -1,25 +1,15 @@
-"""Graph Generation Config Reader
-
-This script reads the config from full_config.yaml file and processes it to be usable.
-It saves config to variables, and converts map strings to python dictionaries.
-"""
-
 import yaml
+from connection_generator import ConnectionGenerator
 
+from config_params.collection_params import CollectionParams
+from config_params.data_integrity_params import DataIntegrityParams
+from config_params.dataset_params import DatasetParams
+from config_params.dataset_to_system_params import DatasetToSystemParams
+from config_params.processing_params import ProcessingParams
+from config_params.system_params import SystemParams
 
-def process_map(config_map, proba=False, enum=False):
-    """Converts string map in format [key1:value1 key2:value2] into a dictionary.
-        key - represents a count of connections (integer) or enums (string).
-        value - represents a count of occurrences (integer) or probability (float).
-
-    Args:
-        config_map: string that contains the map
-        proba: boolean, true if values are probabilities, not counts.
-        enum: boolean, true if keys are enums, not integers.
 
-    Returns:
-        A dictionary of keys and values in converted format.
-    """
+def process_map(config_map, proba=False, enum=False):
```

Comment:
Did you remove the docstring?

Reply:
Yes, by accident. Same with module docstring.